lib/verify.js

```diff
@@ -41,6 +43,8 @@ function verify_metadata(metadata) {
     } else if (object_type === "DictionaryWordObject") {
         assert(!!metadata['word'], "metadata missing word");
         assert(!!metadata['definition'], "metadata missing definition");
+    } else if (object_type === 'VideoObject') {
+        assert(!metadata['contentType'], "Video object should have its contentType set later");
```

Comment:
It's possible that the postal uses this info. @milossymphony do you know if that's the case? I can check too if you don't know.

Reply:
@sgnn7 Not sure I understand the question.  But there is this piece of code when mapping hatch manifest items to content items:
```
  // Decorate the manifest items with some metadata so we can distinguish
  // between them later on
  manifest.assets.forEach(item => {
    item.isHatchAsset = true;
    item.contentType = 'application/x-hatch-asset';
  });

  // This isn't an actual mimetype, but that's fine since our ingestion
  // process is looking for job-system specific values here
  manifest.videos.forEach(item => {
    item.contentType = 'Video';
  });
```

Reply:
Ah, ok - then we're good. I didn't know if the mapping was based on mimeType but it looks like it's based on manifest asset type so I think we're good.